nss-day-cohort-41/tabloidmvc-the-seared-scallops-andy
# added delete functionality for posts

# Description
This branch enables the user to delete a post which they own.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
pull this branch down.  Run the program.  Log in as a user and create a post.  Click the trash can icon on the post and verify that a page appears asking if you are sure if you would like to delete the post.  Click "Delete" and verify that the post has ceased to exist.
# Checklist:
